content/v2.0/influxdb-templates/create.md

```diff
@@ -79,6 +79,34 @@ influx pkg export all \
   -t $INFLUX_TOKEN
 ```
 
+#### Export resources filtered by labelName or resourceKind
+The `influx pkg export all` command has an optional `--filter` flag that filters
+only exports resources that match a specified `labelName` or `resourceKind`.
+Provide multiple filters for both `labelName` and `resourceKind`
+
+###### Export only dashboards and buckets with specific labels
+The following example exports resources that match this predicate logic:
```

Comment:
I think it's important to communicate that this will only export dashboards and buckets. "Specify dashboards and..." feels like it leaves room to assume other resources will get exported as well.

Reply:
Heading may read like there's a way to export only dashboards and buckets without labels--perhaps that's true?

Would "Export dashboards and buckets" work? ...think the "only" part can be communicated in body text? For example, if folks can do something in UI and not in CLI, we'd say "Do xyz in UI" rather than "Do xyz only in UI"